src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public String getRandomBall() {
+        Random random = new Random();
```

Comment:
The ball should be colorized in the method Lottery#getRandomBall.

Method Lottery#info is not needed by task description, you can simply use System.out.println inside Application#main.